modules/gateways/blockonomics/blockonomics.php

```diff
@@ -521,6 +521,13 @@ public function processOrderHash($order_hash, $blockonomics_currency)
         }
         // Process a new order for the id and blockonomics currency
         $new_order = $this->createNewCryptoOrder($order_info, $blockonomics_currency);
+        $log_data = array(
+            'order_id' => $new_order->id_order,
+            'address' => $new_order->addr,
+            'amount' => $new_order->value,
```

Comment:
There is no need to log the amount as this can be found on the invoice itself.

Reply:
Updated